classroom-template-bundle-spec.md

```diff
@@ -69,6 +69,10 @@ role_tags: ['cloud architect', 'developers backend']
 domain_tags: ['infrastructure']
 level: 1
 
+# course_surveys
+course_surveys:
+- 'cloud-training-l2io16cn5i'
```

Comment:
Could this be represented as `course_surveys: ['cloud-training-l2io16cn5i']` because it is just an array of slugs

Reply:
Yep, yaml supports both for arrays -- I can change it if you like!